rcgen/src/lib.rs

```diff
@@ -172,7 +190,25 @@ fn ip_addr_from_octets(octets: &[u8]) -> Result<IpAddr, Error> {
 impl SanType {
 	#[cfg(feature = "x509-parser")]
 	fn try_from_general(name: &x509_parser::extensions::GeneralName<'_>) -> Result<Self, Error> {
+		use x509_parser::der_parser::asn1_rs::{self, FromDer, Tag, TaggedExplicit};
 		Ok(match name {
+			x509_parser::extensions::GeneralName::OtherName(oid, value) => {
```

Comment:
Nit: let's move this arm last, since it's long and probably rarer than the others.